workflows/wf_merlin_magic.wdl

```diff
@@ -159,6 +160,15 @@ workflow merlin_magic {
         samplename = samplename
     }
   }
+  if (merlin_tag == "Staphylococcus aureus") {
+    if (paired_end) {
```

Comment:
@cimendes Can we remove this conditional on `paired_end` ? Spatyper does not require that, correct?